CONTRIBUTE.md

```diff
@@ -172,6 +172,8 @@ It is important to the project that we have benchmarks in place to evaluate the
 8. Pass data as an argument to the function being benched. This will prevent Criterion from including data creation time as part of the benchmark.
 9. Add a profiler see [here](https://github.com/tikv/pprof-rs#integrate-with-criterion) for an example on how to do so with pprof, Criterion, and Flamegraph.
 
+Feel free to use the pls bench as a guideline. Note that it uses functions get get default configurations for profiling and benchmarking. In most cases you can copy and paste such those portions of code. If other configurations are desired it is still easily customizable and explained in the pprof and Criterion documentation.
+
```

Comment:
This needs to be changed to account for the new changes. Do it after everything else is finalized.

Reply:
You can make the change, now that everything's finalized.

Reply:
will do

> Try `Vec::<ArrayView1<&f64>>`

no dice

> Try `Vec::<ArrayView1<&f64>>`